code/modules/hydroponics/hydro_tray_process.dm

```diff
@@ -183,8 +183,6 @@
 				overlays += image(icon = icon, icon_state = "over_lowwater3")
 		else if(get_toxinlevel() < TOXINLEVEL_MAX/5)
 			overlays += image(icon = icon, icon_state = "over_lowwater3")
-		else if(!seed && get_waterlevel() <= WATERLEVEL_MAX/5)
```

Comment:
no more icon for low water if no plant?

Reply:
it's just a redundant else if, it never actually gets triggered ever

Reply:
can't have `if seed && !seed`

Reply:
Right but you CAN move this functionality below the seed check; don't remove the functionality of knowing a tray is low on water without a plant in, that's actually a really impactful hydroponics change. I know, I play a lot of botanist.

Reply:
you're right, hopefully this fixes it